src/dmd/aggregate.d

```diff
@@ -50,6 +50,22 @@ enum Baseok : int
     semanticdone,     // all base classes semantic done
 }
 
+/**
+ * The ClassKind enum is used in AggregateDeclaration AST nodes to
+ * specify the linkage type of the struct/class/interface or if it
+ * is an anonymous class. If the class is anonymous it is also
+ * considered to be a D class.
```

Comment:
As it wasn't obvious why structs need this information, adding a comment here would be nice. Otherwise LGTM.

Reply:
I totally rewrote that comment to include struct... What should it say?
My feeling is, any reason that class needs it is logically transferred to struct too.

Reply:
I didn't rename the enum though, that would have been a very invasive change...
I started renaming it, and then realised how far it spread.

Reply:
>I started renaming it, and then realised how far it spread.

Honestly, I think that it should be renamed even if it's an invasive change. But it can be done in a subsequent PR